backend/hct_mis_api/apps/registration_datahub/tests/test_program_population_to_pending_objects.py

```diff
@@ -371,6 +371,10 @@ def test_create_pending_objects_from_objects(self) -> None:
             pending_document.individual,
             head_of_household_pending_individual,
         )
+        self.assertEqual(
+            pending_document.status,
```

Comment:
pending_document is already `Document.pending_objects.first()` so it was already in this status before the fix and this check doesn't seem to bring anything new